packages/flutter/lib/src/widgets/focus_manager.dart

```diff
@@ -705,18 +705,17 @@ class FocusNode with DiagnosticableTreeMixin, ChangeNotifier {
 
   // Marks the node as dirty, meaning that it needs to notify listeners of a
```

Comment:
This comment seems outdated as nothing is marked as dirty anymore?

Reply:
Updated the comment and added some more context.